appinsights/client.go

```diff
@@ -14,7 +15,7 @@ type TelemetryClient interface {
 	TrackMetricTelemetry(*MetricTelemetry)
 	TrackTrace(string)
 	TrackTraceTelemetry(*TraceTelemetry)
-	TrackRequest(string, time.Time, time.Duration, string, bool)
+	TrackRequest(string, string, string, time.Time, time.Duration, string, bool)
```

Comment:
Why not take argument of RequestTelemetry? We're discussing a similar change to Node.js: https://github.com/Microsoft/ApplicationInsights-node.js/issues/302